src/vivarium_census_prl_synth_pop/tools/cli.py

```diff
@@ -69,14 +72,75 @@ def make_artifacts(
     "-b",
     "--mark-best",
     is_flag=True,
+    help="Marks this version of results with a 'best' symlink.",
 )
 @click.option(
     "-t",
     "--test-run",
     is_flag=True,
+    help="Skips updating the 'latest' symlink with this version of results.",
 )
 @click.option(
-    "-a", "--artifact_path", type=click.Path(exists=True), default=paths.DEFAULT_ARTIFACT
+    "-i",
+    "--artifact-path",
+    type=click.Path(exists=True),
+    default=paths.DEFAULT_ARTIFACT,
+    show_default=True,
+    help="Path to artifact used in simulation.",
+)
+@click.option(
+    "-j",
+    "--jobmon",
+    is_flag=True,
+    help="Launches a jobmon workflow to generate results.",
+)
+@click.option(
+    "-q",
+    "--queue",
+    type=click.Choice(["all.q", "long.q"]),
+    default="all.q",
+    show_default=True,
+    help="NOTE: only required if --jobmon. "
+    "The cluster queue to assign jobmon tasks to. long.q allows for much "
+    "longer runtimes although there may be reasons to send jobs to that queue "
+    "even if they don't have runtime constraints (eg node availability). ",
+)
+@click.option(
+    "-m",
+    "--peak-memory",
+    type=int,
+    default=200,
+    show_default=True,
+    help="NOTE: only required if --jobmon. "
+    "The estimated maximum memory usage in GB of an individual simulate job. "
+    "The simulations will be run with this as a limit. ",
+)
+@click.option(
+    "-r",
+    "--max-runtime",
+    type=str,
+    default="4:00:00",
+    show_default=True,
+    help="NOTE: only required if --jobmon. "
+    "The estimated maximum runtime ('hh:mm:ss') of the jobmon tasks. "
+    "Once this time limit is hit, the cluster will terminate jobs regardless of "
+    "queue. The maximum supported runtime is 3 days. Keep in mind that the "
+    "session you are launching from must be able to live at least as long "
+    "as the simulation jobs, and that runtimes by node vary wildly. ",
+)
+@click.option(
```

Comment:
Don't give the user a choice. They should always be using `proj_simscience_prod`

Reply:
Are you sure? What abvout when debugging? I've been doing all this development on `proj_simscience`, eg.

Reply:
Will either remove the option altogether or change it to the default